include/hip/hip_runtime_api.h

```diff
@@ -696,6 +717,7 @@ enum hipLimit_t {
 // Stream per thread
 /** Implicit stream per application thread.*/
 #define hipStreamPerThread ((hipStream_t)2)
+#define hipStreamLegacy ((hipStream_t)3)
```

Comment:
why is this needed?

Reply:
It's defined for CUDA and gives you access to the legacy default queue. Why isn't it defined for AMD?